src/main/java/uk/ac/cam/cl/dtg/teaching/pottery/api/RepoController.java

```diff
@@ -85,6 +93,16 @@ RepoInfo makeRemoteRepo(
   List<String> listTags(@PathParam("repoId") String repoId)
       throws RepoStorageException, RepoNotFoundException;
 
+  @GET
+  @Path("/{repoId}/problemStatement")
+  @Produces("text/html")
+  @ApiOperation(
+      value = "Get the problem statement for this parameterisation of the task.",
+      response = String.class)
+  String getParameterisedProblemStatement(@PathParam("repoId") String repoId)
```

Comment:
I think you could just call this getProblemStatement - its implicitly parameterised because its tied to the repo

Reply:
will rename